io/module_write_netcdf.F90

```diff
@@ -9,7 +9,7 @@
 !> Return error to ESMF and finalize it.
 #define NC_ERR_STOP(status) \
     if (status /= nf90_noerr) write(0,*) "file: ", __FILE__, " line: ", __LINE__, trim(nf90_strerror(status)); \
-    if (status /= nf90_noerr) call ESMF_Finalize(endflag=ESMF_END_ABORT)
+    if (status /= nf90_noerr) call ESMF_Finalize(endflag=ESMF_END_ABORT) !< ???
```

Comment:
This define already has doxygen documentation just above it. So it does not need the !< ???